static/app/views/organizationStats/usageChart/index.tsx

```diff
@@ -109,6 +110,8 @@ export enum SeriesTypes {
   ACCEPTED = 'Accepted',
   DROPPED = 'Dropped',
   PROJECTED = 'Projected',
+  RESERVED = 'Reserved',
+  ON_DEMAND = 'OD Spent',
```

Comment:
We try to avoid billing concepts in `sentry`. Is it possible to extend this enum in `getsentry`?